kalite/control_panel/tests/control_panel.py

```diff
@@ -551,60 +557,60 @@ def test_device_log_csv_endpoint(self):
         self.assertEqual(len(rows), 2, "API response incorrect")
 
 
-class CSVExportBrowserTests(CSVExportTestSetup, BrowserActionMixins, CreateAdminMixin, KALiteBrowserTestCase):
+# class CSVExportBrowserTests(CSVExportTestSetup, BrowserActionMixins, CreateAdminMixin, KALiteBrowserTestCase):
```

Comment:
I get what you're saying, my concern is that the tests _won't_ be re-implemented, and then we'll just have a bunch of useless old code sitting around. I suggest one of:
1. Re-implementing these tests immediately. The obvious concern is that we have other equally as important/urgent stuff to do. (Do you have the time and inclination to do this?)
2. We set a date when someone will begin re-implementing them, so it doesn't languish.
3. We remove the tests entirely and don't give it a second thought until we actually encounter a bug, then we re-implement them. Obviously this is a gamble.

My contention is that just commenting them out is essentially the same as choosing 3.